src/server/mod.rs

```diff
@@ -279,6 +277,11 @@ impl<State: Send + Sync + 'static> Server<State> {
     ///
     /// This lower-level method lets you host a Tide application within an HTTP
     /// server of your choice, via the `http_service` interface crate.
+    ///
+    /// # Middleware
+    ///
+    /// This method does not enable default middleware. See `into_mock` for
+    /// converting Tide into a server that can be used for testing purposes.
     pub fn into_http_service(self) -> Service<State> {
```

Comment:
How about leaving this as is, and provide a new method that doesn't apply any default middleware? One rarely calls `.into_http_service()` directly for nesting since `Route::nest` handles it instead, so I think it should be okay?